cosmos/providers/dbt/parser/project.py

```diff
@@ -157,28 +149,24 @@ def __post_init__(self) -> None:
                                 else set()
                             )
 
-        # set the config and set the parsed file flag to true
         self.config = config
 
-    def _extract_config(self, kwarg, config_name: str):
+    def _extract_config(self, kwarg: Keyword, config_name: str):
         if hasattr(kwarg, "key") and kwarg.key == config_name:
             try:
-                # try to convert it to a constant and get the value
                 value = kwarg.value.as_const()
-                if isinstance(value, List):
-                    value = [f"{config_name}:{item}" for item in value]
-
-                if isinstance(value, str):
-                    value = [f"{config_name}:{value}"]
-
-                return value
-
-            except Exception as e:
-                # if we can't convert it to a constant, we can't do anything with it
+            except Impossible as e:
                 logger.warning(
                     f"Could not parse {config_name} from config in {self.path}: {e}"
                 )
                 pass
+            else:
+                parsed_value = None
+                if isinstance(value, List):
```

Comment:
I think this should be `list`, right?
```suggestion
                if isinstance(value, list):
```